aws/ec2/snappy_ec2.py

```diff
@@ -331,6 +333,16 @@ def parse_args():
     parser.add_option(
         "--instance-profile-name", default=None,
         help="IAM profile name to launch instances under")
+    parser.add_option(
+        "--assume-role-arn", type="string", default=None,
+        help="ARN of assumed role to launch a instance")
+    parser.add_option(
+        "--assume-role-timeout", type="int", default=3600,
+        help="Session timeout of the assumed role, min is 900 seconds and max is 3600 seconds")
```

Comment:
Suggested help text: "Timeout in seconds for the temporary credentials of the assumed IAM role."

BTW, is the limit (900 - 3600) enforced?

Reply:
yes time limit is 900-3600 enforced in AWS API itself.